packages/server/src/jsRunner/bundles/index.ts

```diff
@@ -8,20 +8,21 @@ export const enum BundleType {
 }
 
 const bundleSourceCode = {
-  [BundleType.HELPERS]: "../bundles/index-helpers.ivm.bundle.js",
-  [BundleType.BSON]: "../bundles/bson.ivm.bundle.js",
+  [BundleType.HELPERS]: "./index-helpers.ivm.bundle.js",
+  [BundleType.BSON]: "./bson.ivm.bundle.js",
 }
 
 export function loadBundle(type: BundleType) {
-  if (environment.isJest()) {
+  if (!environment.isBundled) {
```

Comment:
I assume this is only set when running it through esbuild?

Reply:
I was not sure if this is the best name...

Reply:
Ok, refactor and not needed :)